shared/constants/types/rpc-gen.js.flow

```diff
@@ -0,0 +1,4103 @@
+// @flow strict
```

Comment:
Run `make clean` before `make` to avoid generating these files. Or use `yarn update-protocol`

Reply:
thanks

Reply:
Is running protocol's `make` and not `yarn update-protocol` no longer cool?

Reply:
`make` is a little less cool; update-protocol updates FE generated stuff so CI doesn't have to tell you to

Reply:
A suggestion for `update-protocol` would be to use `make` instead of `make clean && make`, since the point of `make` is to be faster